cypress/e2e/token.test.ts

```diff
@@ -0,0 +1,20 @@
+import { getTestSelector, getTestSelectorStartsWith } from '../utils'
+
+describe('Testing tokens on uniswap page', () => {
+  before(() => {
+    cy.visit('/')
+  })
+
+  it('should load token leaderboard', () => {
+    cy.visit('/tokens/ethereum')
+    cy.get(getTestSelectorStartsWith('token-table')).its('length').should('be.gte', 25)
```

Comment:
why gte?

Reply:
I don't want to block the frontend if backend is failing to return 100 assets

Reply:
hm yea i see